docker-compose.yml

```diff
@@ -68,6 +68,7 @@ services:
       FEATURE_REDIS_SESSION_STORE: 'true'
       FEATURE_SAVE_AND_EXIT_OPTION: 'false'
       FEATURE_SHOW_AFFECTED_AREA_SECTION: 'false'
+      FEATURE_HIDE_ROUTE: 'true'
```

Comment:
Can this be a specific feature name that is activated or inactive. Something like  
FEATURE_PROJECT_TIMELINE_LINK = 'false'

is perhaps more readable and consistent with other feature flags.

Reply:
Makes sense. I've tried to add a broader tag for possible future hiden routes too...